command/agent/agent.go

```diff
@@ -458,6 +458,15 @@ func (a *Agent) RPC(method string, args interface{}, reply interface{}) error {
 	return a.client.RPC(method, args, reply)
 }
 
+// RPCRetry is used to make an RPC call to the Consul servers.  This allows
+// the agent to implement the Consul.Interface.
+func (a *Agent) RPCRetry(method string, args interface{}, reply interface{}, timeout time.Duration) error {
```

Comment:
It seems confusing that this `RPCRetry` takes a duration and the other `RPCRetry` takes a deadline. How about a deadline here, too?